ci/upload_gcs_artifact.sh

```diff
@@ -18,8 +18,8 @@ if [ ! -d "${SOURCE_DIRECTORY}" ]; then
   exit 1
 fi
 
-if [[ "$TARGET_SUFFIX" == "docs" ]]; then
-    # docs upload to the last commit sha (first 7 chars) in the developers branch
+if [[ "$BUILD_SOURCEBRANCHNAME" != "master" ]]; then
```

Comment:
This would still break release branches for etc. Just use BUILD_REASON to see if that's a PullRequest? Or check if HEAD is a merge commit or not.

Reply:
lets go with `BUILD_REASON`, im unclear whether there could be merge commits and it still be a pull request